pyproject.toml

```diff
@@ -15,7 +15,7 @@ classifiers = ["Topic :: Software Development"]
 
 # Requirements
 dependencies = [
-    "vyper >= 0.3.7",
+    "vyper @ git+https://github.com/vyperlang/vyper.git@master",
```

Comment:
this should always be pinned to a specific commit, if upstream breaks something here, boa will not be broken.